spec/rails_helper.rb

```diff
@@ -6,6 +6,11 @@
 require 'spec_helper'
 require 'rspec/rails'
 require 'shoulda-matchers'
+require 'capybara/poltergeist'
+Capybara.javascript_driver = :poltergeist
+
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.